Dynatrace/dynatrace-operator
# Add CSI Driver label for to support OCP 4.13 PodSecurityStandards

# Description

Support for [CSI Inline Epehemeral Volume Security](https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-inline-vol-security.md) feature, by setting the required label on the CSIDriver resource, in order to get restricted PSS on customer namespaces.

## How can this be tested?
make install on an Openshift cluster, should install the CSI Driver with the correct Label (note: not the Pod or daemonset, but the CSI ressource should have the label)


## Checklist
